docs/overview.rst

```diff
@@ -85,6 +85,11 @@ Two ``linker`` functions are made available:
 
    Links a `contract_name` to a `linked_type`. The `linked_type` must have already been deployed.
 
+.. py:method:: do_callback(callback_fn)
```

Comment:
Possible alternate method name borrowed from the [django migrations documentation](https://docs.djangoproject.com/en/2.1/ref/migration-operations/#django.db.migrations.operations.RunPython) is to use the name `run_python`.  I don't have a strong opinion.

Reply:
sounds good, made the change to follow in their footsteps